xil_cloud/docker-compose.yml

```diff
@@ -416,10 +416,23 @@ services:
     volumes:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ./evc_sumo/evc_sumo_cfg.json:/home/carma/src/resources/evc_sumo_cfg.json
-    depends_on: 
+    depends_on:
       - carma-simulation
     command: bash -c "sleep 10;x-terminal-emulator -e python3.8 evc_sumo_bridge.py --asc3app-path '../evcfile/asc3app-application.zip' --traci-ip 172.2.0.2"
 
+  scenario-runner:
+    image: usdotfhwastoldev/carma-scenario-runner:latest
```

Comment:
- let's rename it to `usdotfhwastoldev/carma-scenario-runner:develop` and manually push the image to the dockerhub. So it is available if needed.
- We should probably comment it out for now not not interrupt other people's testing? Such as telematics team. And only people who need it can turn it on if they wanted to

Reply:
Should we comment it out for both `xil_cloud` and `xil_cloud_telematics`?